worker/src/scripts/verifyClickhouseRecords/index.ts

```diff
@@ -527,7 +523,8 @@ async function verifyClickhouseObservation(postgresObservation: any) {
       case "prompt_tokens": {
         if (
           pgValue !== 0 &&
-          pgValue !== (clickhouseRecord as any)["input_usage_units"]
+          pgValue.toString() !==
```

Comment:
Why is tostring required here?

Reply:
Oh that may be obsolete. In the old incarnation zod coerced those values into strings for UInt64. I'll review this script when we run it again.